flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/parser/TransformParserTest.java

```diff
@@ -292,6 +292,9 @@ public void testTranslateFilterToJaninoExpression() {
         testFilterExpression("cast(null as decimal)", "castToBigDecimal(null, 10, 0)");
         testFilterExpression("cast(null as char)", "castToString(null)");
         testFilterExpression("cast(null as varchar)", "castToString(null)");
+        testFilterExpression(
+                "cast(CURRENT_TIMESTAMP as TIMESTAMP)", "castToTimestamp(__time_zone__)");
+        testFilterExpression("cast(dt as TIMESTAMP)", "castToTimestamp(__time_zone__)");
```

Comment:
Expect value may be incorrect.
```
castToTimestamp(currentTimestamp(__epoch_time__), __time_zone__)
castToTimestamp(dt, __time_zone__)
```

Reply:
sorry, I forgot update here, fix it soon